.github/labels.yml

```diff
@@ -0,0 +1,31 @@
+- name: status/possible-priority
```

Comment:
Are those labels used across the org? Might be worth checking if labels are supported in an org-wide `.github` repo.

Reply:
We do this for builders (and similarly for buildpacks): https://github.com/paketo-buildpacks/github-config/blob/main/builder/.github/labels.yml
We could do the same for stacks.

Reply:
Seems like it's possible to set labels org-wide: https://docs.github.com/en/organizations/managing-organization-settings/managing-default-labels-for-repositories-in-your-organization

Seems like this would have to be done by someone with org admin permissions.